src/panel-triggers/components/AckModal.tsx

```diff
@@ -39,7 +39,7 @@ export interface AckProblemData {
 }
 
 const severityOptions = [
-  {value: 0, label: 'Not classified'},
+  //{value: 0, label: 'Not classified'},
```

Comment:
What effects does this change have? If nothing, maybe it should be reverted so that the code is similar to the original one (less merge problems in the future).

Reply:
This constant isn't even used (or so it seems) so it's unnecessary (probably), but I removed the comment.